docs/installation/docker/site-template/customizations.md

```diff
@@ -0,0 +1,49 @@
+# Customizations
+
+The following sections describe optional customizations you can make to further modify your Islandora site.
+
+## Blocking Connections
+
+### Blocking IP Addresses
+
+#### iptables
+
+If your server uses iptables, you can block a range of IP addresses using the following command
+
+`iptables -I DOCKER-USER -s XXX.XXX.XXX.0/24 -j DROP`
+
+### Blocking by User Agent
+
+You can modify how nginx handles certain user agents by modifying `/etc/nginx/shared/drupal.defaults.conf` inside your Drupal container. For example, by adding:
```

Comment:
I think we should add some guidance here on how to do this.

We should utilize [buildkit's nginx conf](https://github.com/Islandora-Devops/isle-buildkit/blob/bd54c1a1070ab92ddc5bc63e0e062fbfc754a4a2/nginx/rootfs/etc/confd/templates/nginx.conf.tmpl#L168) that includes rules from any `*.conf` file in `/etc/nginx/http.d/` in the drupal container. So IMO the guidance should be to bind mount a new file in that location and/or add that to your Dockerfile build

Reply:
out of scope of this PR